stdlib/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""A simple package that returns the path to the stdlib stubs folder."""
+
+__version__ = '1.0a1'
+
+import pathlib
+
+def get_stdlib_dir() -> pathlib.Path:
+    return pathlib.Path().absolute()
```

Comment:
`Path()` returns the cwd; don't you want something like `Path(__file__).parent`?

Reply:
Ah yes, I meant to do that.